src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -92,6 +92,13 @@ public function load(array $configs, ContainerBuilder $container)
             $dir = dirname($reflection->getFileName()).'/Resources/views';
             if (is_dir($dir)) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $bundle);
+
+                $classMock = new $class();
+                $parent = $classMock->getParent();
+
+                if ($parent) {
```

Comment:
The only way I could think of making this work is indeed, recursively instantiating the parent class, getting the parent bundle name, retrieve its class from the list of bundles, and so on. Is that what you would recommend?